pkg/virt-controller/network/multus_annotations.go

```diff
@@ -36,6 +36,11 @@ import (
 	virtconfig "kubevirt.io/kubevirt/pkg/virt-config"
 )
 
+type IPAMClaimParams struct {
+	ClaimName   string
+	NetworkName string
+}
+
```

Comment:
maybe move this to the new `ipamclaims` pkg ? Under a types file.

Reply:
It can be done.

Please check out this [patch](https://paste.centos.org/view/527e7bc2). Will expire tomorrow.

Not sure it is worth it, but definitely it enabled removing some dependencies. 

I'll keep experimenting about the multus_annotations part.

Reply:
If you can please provide a patch that can be applied on top of current PR i will be glad,
cant apply the patch in the link, and it seems from glance that logic was changed or prone to be changed
will be much easier if i can apply it please to check

Reply:
sorry i didnt manage to apply the patch even if i jump to the one before latest push
(unless i did something wrong)
since it has lots of changes and delicate, if you still want me to do it, please kindly either give guidelines or rebase so i will manage to patch it

Reply:
I give up.

Reply:
I am going to work on https://github.com/kubevirt/kubevirt/pull/11410#issuecomment-2100518858 on a follow-up PR that i am starting right away as discussed
once the code is changed it might be more straight forward
meanwhile if needed i can add `ipamclaims_types`